prog/dftb+/lib_extlibs/negf_int.F90

```diff
@@ -993,12 +994,13 @@ end subroutine negf_current
 
   !> Calculates density matrix with Green's functions
 #:if WITH_MPI
-  subroutine calcdensity_green(iSCCIter, mpicomm, groupKS, ham, over, iNeighbor, nNeighbor,&
+  subroutine calcdensity_green(iSCCIter, enecomm, kscomm, groupKS, ham, over, iNeighbor, nNeighbor,&
       & iAtomStart, iPair, img2CentCell, iCellVec, cellVec, orb, kPoints, kWeights, mu, rho, Eband,&
       & Ef, E0, TS)
 
     !> MPI communicator
-    type(mpifx_comm), intent(in) :: mpicomm
+    type(mpifx_comm), intent(in) :: enecomm
```

Comment:
What does the enecomm name mean? It suggests a communicator between energies, but looking at the call is the intra-group communicator for a specific (k,s) matrix.

Reply:
Exactly, it is a communicator dealing with energy points distribution for a given k,s.
On this communicator I can make reduce operations across energy points, hence this name